dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/vsme/custom/SmeSimpleCustomComponentBase.kt

```diff
@@ -0,0 +1,69 @@
+package org.dataland.frameworktoolbox.frameworks.vsme.custom
+
+import org.apache.commons.text.StringEscapeUtils
+import org.dataland.frameworktoolbox.intermediate.FieldNodeParent
+import org.dataland.frameworktoolbox.intermediate.components.ComponentBase
+import org.dataland.frameworktoolbox.intermediate.components.addStandardCellWithValueGetterFactory
+import org.dataland.frameworktoolbox.intermediate.components.addStandardUploadConfigCell
+import org.dataland.frameworktoolbox.intermediate.components.requireDocumentSupportIn
+import org.dataland.frameworktoolbox.intermediate.datapoints.NoDocumentSupport
+import org.dataland.frameworktoolbox.specific.fixturegenerator.elements.FixtureSectionBuilder
+import org.dataland.frameworktoolbox.specific.uploadconfig.elements.UploadCategoryBuilder
+import org.dataland.frameworktoolbox.specific.viewconfig.elements.SectionConfigBuilder
+import org.dataland.frameworktoolbox.specific.viewconfig.elements.getTypescriptFieldAccessor
+import org.dataland.frameworktoolbox.specific.viewconfig.functional.FrameworkDisplayValueLambda
+import org.dataland.frameworktoolbox.utils.typescript.TypeScriptImport
+
+/**
+ * Represents the SmeSimpleCustomComponentBase component
+ * @param viewFormattingFunctionName the name of the formatting function for the view page in SmeDisplayValueGetters.ts
+ */
+open class SmeSimpleCustomComponentBase(
```

Comment:
do you want to keept the class names as sme... and not vsme?

Reply:
Sollen am Ende alle vsme heißen. Soweit ich gesehen hab ist das hier auch korrgiert worden